examples/workflow/3_perform_fit.ipynb

```diff
@@ -0,0 +1,375 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "estimator, initial_parameters = \\\n",
+    "    pwa.create_unbinned_log_likelihood_function_tree_estimator(\n",
+    "        intensity, data_set)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Notice that you not only receive a estimator object, but also a list of fit parameters. You use this list of fit parameters to initialize the optimizer later on.\n",
+    "\n",
+    "The parameters contains the following info:\n",
+    "- initial value\n",
+    "- whether they are free or fixed\n",
+    "- optional upper and lower limits\n",
+    "- errors that give certain optimizers hints on the step size"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "initial_parameters"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3.2 Modify parameters"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The fit parameters in this list have been initialized with the values stated in the XML file or with default values if unspecified, but they can easily be modified in Python as well:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(\"\\nthis parameter is initially not fixed:\")\n",
+    "print(initial_parameters[8])\n",
+    "\n",
+    "initial_parameters[7].is_fixed = True\n",
+    "initial_parameters[8].is_fixed = True\n",
+    "print(\"\\nand now it is fixed:\")\n",
+    "print(initial_parameters[8])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To make the fit process a bit more interesting, we modify one of the parameters to a different initial value than the true value:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "original_value = initial_parameters[12].value\n",
+    "initial_parameters[12]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "initial_parameters[12].value = 2.0\n",
+    "initial_parameters[12]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3.3 Optimize the intensity model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now it's time to perform the fit! This is quite simple: just create an optimizer instance of your choice, here [Minuit2](https://root.cern.ch/doc/master/Minuit2Page.html), and call its `optimize` method to start the fitting process."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The computation time depends on the complexity of the mode, number of data events, the size of the phase space sample, and the number of free parameters."
```

Comment:
*complexity of the model

Reply:
https://github.com/ComPWA/pycompwa/pull/74/commits/c8a49675e989f3ec982584cbd916812bc948dccf